packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -46,15 +46,12 @@ export const StyledSideNavListItem = styled("li", {
   backgroundColor: "unset",
   borderRadius: odysseyDesignTokens.BorderRadiusMain,
   transition: `backgroundColor ${odysseyDesignTokens.TransitionDurationMain}, color ${odysseyDesignTokens.TransitionDurationMain}`,
+  marginBlock: "2px",
```

Comment:
Don't think we need this

Reply:
I saw an issue due to the `box-shadow` with 2px usage for `focus-visible`. If an item is focused and if the user hovers over that item, box-shadow disappears. It's hard of hard to explain. I can demo it if you would like to see it. So I readjusted the overall spacing based on the new Drag-n-drop component layout

Reply:
example:
<img width="310" alt="image" src="https://github.com/user-attachments/assets/19f9260d-d692-4dc9-b92e-88d127606ad5">
<img width="310" alt="image" src="https://github.com/user-attachments/assets/7bfbcac5-82be-42ee-9d5c-714235df1435">

with the fix..
<img width="323" alt="image" src="https://github.com/user-attachments/assets/c979af41-6b44-4867-a1c0-013ad30e53fd">

Reply:
This looks like a different issue. We shouldn't be adding margin or padding that's not in the Figma mock.

I'm sure there's a way to make sure that border shows up. I think the selected item has its own background color when hovered or focused.

Reply:
I think that is correct. Seems like the draggable item just needs to maybe not have any hover styles. I'd have to pull your branch down and mess around with it. Which, I'm happy to do if you'd like